pkg/config/config_test.go

```diff
@@ -920,6 +920,62 @@ func TestDeleteApplication(t *testing.T) {
 	}
 }
 
+func TestGetTimeout(t *testing.T) {
+	tempConfigFile, err := ioutil.TempFile("", "odoconfig")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer tempConfigFile.Close()
+	os.Setenv(configEnvName, tempConfigFile.Name())
+	zeroValue := 0
+	nonzeroValue := 5
+	tests := []struct {
+		name           string
+		existingConfig Config
+		want           int
+	}{
+		{
+			name:           "validating value 1 from config in default case",
```

Comment:
you should have "Case 1, Case 2, etc." for the names, similar format as the other tests we have.